resources/vbmc/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7
+
+RUN apt update && \
+  apt install -y libvirt0 libvirt-dev && \
+  pip3 install virtualbmc
```

Comment:
nit: if you use centos as a base for this container, you can use python-virtualbmc from RDO.

Reply:
does this bring something compared to pip installation ?